tests/components/venstar/test_init.py

```diff
@@ -36,6 +36,9 @@ async def test_setup_entry(hass: HomeAssistant):
     ), patch(
         "homeassistant.components.venstar.VenstarColorTouch.update_alerts",
         new=VenstarColorTouchMock.update_alerts,
+    ), patch(
+        "homeassistant.components.venstar.VenstarColorTouch.get_runtimes",
+        new=VenstarColorTouchMock.get_runtimes,
     ), patch(
         "homeassistant.components.onewire.sensor.asyncio.sleep"
```

Comment:
Please remove this patch.

Reply:
Why am I removing this patch?  get_runtimes in the api makes a direct call to requests and triggers the pytest socket error.

Reply:
We're not supposed to patch the onewire integration in the venstar integration. That looks completely wrong.